### DISCUSSION TITLE
CHGNet in pyiron

### DISCUSSION THREAD

**Discussion by Joachim-S160** [pyiron/pyiron#1528] [Ideas]
Recently, I saw that a few software developers made it possible to use CHGNet, a universal MLIP, in LAMMPS (https://github.com/CederGroupHub/chgnet/issues/57). I was wondering if there already exists some type of workflow to use MLIPs such as CHGNet in pyiron. If this is the case, feel free to lead me to the documentation. If not, I believe this could be quite interesting.

**Comment by pmrv** [comment]
Once you have compiled lammps with the chgnet interface, you can configure pyiron to use it [here](https://pyiron.readthedocs.io/en/latest/source/installation.html#custom-executables-and-parameter-files) and to use a custom potential file instead of the default pair_style database, see [here](https://pyiron.readthedocs.io/en/latest/source/faq.html#how-do-i-install-additional-codes-for-pyiron).

EDIT: Iirc, chgnet and friends also come with ASE calculators.  We have the capability to use any calculators for atomistic simulations with [this job](https://github.com/pyiron/pyiron_atomistics/blob/6ea4e3eb2a4df6f645f705e9accd2f7513565418/pyiron_atomistics/gpaw/pyiron_ase.py#L26).  It's not documented well, but it should be usable in the same way, as say a Lammps job.
